src/icons/navigator/Home.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+
+interface Props {
+  color?: string
+  size?: number
+}
+
+const Home = ({ color, size }: Props) => (
+  <Svg width={size} height={size} viewBox="0 0 14 16" fill="none">
```

Comment:
similar question here. could you have a 20x20 sized svg, so you also don't need to override the size explicitly in TabNavigator

Reply:
@satish-ravi sure, will do!

Reply:
@satish-ravi fixed!